.github/workflows/coverage-ratchet.yml

```
@@ -0,0 +1,243 @@
+      run: mix credo --strict
+
+    - name: Setup test database
+      env:
+        DATABASE_URL: postgres://postgres:postgres@localhost:5432/eve_tracker_test
+      run: |
+        mix ecto.create
+        mix ecto.migrate
+
+    - name: Get baseline coverage
+      id: baseline
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:8:30: Double quote to prevent globbing and word splitting

(shellcheck)